azure/sdk-identity-resources-storage/README.md

```diff
@@ -0,0 +1,71 @@
+﻿## Prerequisites
+
+* An [Azure subscription]().
+
+## Preparing to run the sample
+
+Use the [Azure Cloud Shell](https://shell.azure.com) to create/get client secret credentials.
+
+* Create a service principal and configure its access to Azure resources:
+
+    ```bash
+    az ad sp create-for-rbac -n <your-application-name> --skip-assignment
+    ```
+
+    Output:
+
+    ```json
+    {
+        "appId": "generated-app-ID",
+        "displayName": "dummy-app-name",
+        "name": "http://dummy-app-name",
+        "password": "random-password",
+        "tenant": "tenant-ID"
+    }
+    ```
+
+* Use the returned credentials above to set the following environment variables:
+
+    |variable name|value
+    |-|-
```

Comment:
Looks like you're missing the closing pipe for each row in this table. It might also be good to alphabetize the table rows by variable name.